### PR TITLE
Feature/item expanded page

### LINKED ISSUE #10: Component | Single Craft
- [ ] structure, plan and style single craft component

### LINKED ISSUE #16: User Story | Single Item
- [ ] As a user, I should be able to view an item

### PR DESCRIPTION
**Pull request by Abumpus1**
## What was changed:
- Adds ItemExpanded page
- App now holds state for item data
- App passes down item data as props

## Explain the reason for changes:
- Move state to App to allow for single query of data
- Made single page to copy wireframe

## Link to issue:
closes #10 
closes #16

### CODE REVIEW

File: src/Components/ItemCard.js [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
     return (
-        <ItemCardSection className="item-card">
-           <ItemCardImg className="stock-img" src={require("../assets/myrlene-numa-SnITZTTeJVE-unsplash.jpg")} alt="craft-item" />
-           <ItemBoxDiv>
-                <ItemH4>{name}</ItemH4>
-                <ItemP>Amount: {amount}</ItemP>
-                <ItemP>Crafty Joe</ItemP>
-           </ItemBoxDiv>
-        </ItemCardSection>
+        <Link style={{textDecoration: "none"}} to={`/craft/${id}`}>
```
Comment (Wperricone) [inline src/Components/ItemCard.js:8]: I'm glad these changes still worked with the testing

File: src/Components/ItemExpanded.js [hunk 2]
```diff
@@ -1,16 +1,101 @@
...
+                    </TitleAmountDiv>
+                    <DescP>{item.description}</DescP>
+                    {/* <button className="contact-button"><a href="mailto:someone@example.com">Contact Crafter</a></button> */}
+                    <Button name="Contact Crafter" link="" />
+                </ItemExpandedDiv>
+            </>
+            : <h1>Loading...</h1>}
+        </ItemExpandedSection>
+    )
 }
```
Comment (stephanie-roe) [inline src/Components/ItemExpanded.js:36]: Love the styling and structure of this piece. Great job implementing a bit of error handling. I think in the future, we can implement error handling with a sad path (no search) and maybe use a boolean to toggle if the content is just loading as you mentioned.

**Review by stephanie-roe — APPROVED** [review]
Looking great!!

File: src/Components/ItemExpanded.js [hunk 3]
```diff
@@ -1,16 +1,101 @@
...
+                    <TitleAmountDiv>
+                        <H2>{item.name}</H2>
+                        <AmountP>Amount: {item.amount}</AmountP>
+                    </TitleAmountDiv>
+                    <DescP>{item.description}</DescP>
+                    {/* <button className="contact-button"><a href="mailto:someone@example.com">Contact Crafter</a></button> */}
+                    <Button name="Contact Crafter" link="" />
+                </ItemExpandedDiv>
+            </>
+            : <h1>Loading...</h1>}
```
Comment (JoeRupp) [inline src/Components/ItemExpanded.js:33]: Nice error handling here. We'll expand on this later in case the user inputs a craft id that doesnt exist

**Pull request merged**